src/index.ts

```diff
@@ -57,3 +61,4 @@ export const edkt = {
 
 // This will expose the exported audiences & allow tree shaking
 export * from './audiences';
+export * from './store';
```

Comment:
Why are we exporting `*` from store here?

Reply:
So consumers of edkt like the wrapper have access to the stores for caching, matched audiences & page views